cfme/containers/provider/openshift.py

```diff
@@ -25,17 +25,19 @@ def __init__(self, name, value, field_type=None, href=None):
 
 class OpenshiftDefaultEndpoint(ContainersProviderDefaultEndpoint):
     """Represents Openshift default endpoint"""
+
     @staticmethod
-    def get_ca_cert():
+    def get_ca_cert(connection_info):
         """Getting OpenShift's certificate from the master machine.
         Args:
-           No args.
+            connection_info (dict): username, password and hostname for OCP
         returns:
             certificate's content.
         """
-        cert_file_path = path.join(str(data_path), 'cert-auths', 'cmqe-tests-openshift-signer.crt')
-        with open(cert_file_path) as f:
-            return f.read()
+
+        with ssh.SSHClient(**connection_info) as provider_ssh:
+            _, stdout, _ = provider_ssh.exec_command("cat /etc/origin/master/ca.crt")
+            return str("".join(stdout.readlines()))
```

Comment:
__OPTIONAL__
I think "".join(stdout.readlines()) is enough right?

Reply:
I wasn't sure if the appliance approve the cert in Unicode so I forced conversion to ASCII